src/seedu/addressbook/Main.java

```diff
@@ -111,7 +112,9 @@ private CommandResult executeCommand(Command command)  {
             CommandResult result = command.execute();
             storage.save(addressBook);
             return result;
-        } catch (Exception e) {
+        }catch (IOException e){
+          return new CommandResult(e.getMessage());
```

Comment:
As discussed in tutorial, please do not just provide the user with a generic error message or stack trace but elaborate more on what went wrong.